samples/subsys/dap/boards/rpi_debug_probe.overlay

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The Zephyr Project Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+		zephyr,shell-uart = &cdc_acm_uart0;
+	};
+
+	uart-bridge0 {
+		compatible = "zephyr,uart-bridge";
+		peers = <&cdc_acm_uart1 &uart1>;
+	};
+
+	dp0 {
+		compatible = "zephyr,swdp-gpio";
+		status = "okay";
+		clk-gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+		dio-gpios = <&gpio0 14 GPIO_PULL_UP>;
+		port-write-cycles = <2>;
```

Comment:
I think it should be `<1>`, based on what I read in the datasheet. But I am not sure what the GPIO driver actually does.

Reply:
alright changed